stravalib/client.py

```diff
@@ -508,7 +416,7 @@ def get_club(self, club_id):
         :rtype: :class:`stravalib.model.Club`
         """
         raw = self.protocol.get("/clubs/{id}", id=club_id)
-        return model.Club.parse_obj(raw)
+        return model.Club.parse_obj({**raw, **{"bound_client": self}})
```

Comment:
This is how we should add the bound client to an entity when deserializing it. When dropping python 3.8 support in the future, this can be expressed cleaner as `raw | {'bound_client: self}`